app/assets/images/payment_icons/multibanco.svg

```diff
@@ -0,0 +1 @@
 01-.102.073l-.011.005-.01.007H16.07a.608.608 0 01-.174.032h-.042a.593.593 0 01-.193-.04h-.05l-.01-.006-.009-.005h-.005a.538.538 0 01-.093-.062l-.02-.017-.014-.015-.036-.009-.017-.02a.66.66 0 01-.063-.093v-.006l-.005-.008-.006-.01v-.044L13.591 8.86l-.484 4.45a.59.59 0 11-1.174-.123l.572-5.248a.98.98 0 01.844-.862h.03c.051-.003.1-.003.15 0a.97.97 0 01.865.617l1.462 3.696 1.459-3.708a.97.97 0 01.866-.617.939.939 0 01.15 0h.03a.978.978 0 01.844.866zM13.792 20.371a.157.157 0 01-.209.176.16.16 0 01-.105-.139l-.134-1.136-.482 1.115a.158.158 0 01-.207.083.152.152 0 01-.082-.083l-.482-1.115-.136 1.136a.157.157 0 11-.312-.037l.158-1.319a.257.257 0 01.139-.196l.031-.015h.012a.273.273 0 01.298.097l.019.029c.002.007.005.012.009.019l.409.947.408-.956a.257.257 0 01.185-.149.793.793 0 01.048-.007h.048a.265.265 0 01.196.126.154.154 0 01.021.045.16.16 0 01.01.031v.02zM17.135 20.389a.158.158 0 01-.317 0v-1.246h-.449a.158.158 0 01-.136-.237.154.154 0 01.136-.079h1.214c.121 0 .197.132.137.237a.163.163 0 01-.137.079h-.448zM18.119 20.389a.157.157 0 01-.236.136.154.154 0 01-.08-.136v-1.403c0-.123.131-.199.237-.137a.16.16 0 01.079.137zM21.352 20.371a.158.158 0 01-.227.153.164.164 0 01-.086-.115l-.056-.446h-.542a.159.159 0 01-.155-.226.157.157 0 01.155-.089h.507l-.014-.116v-.012c-.004-.026-.012-.053-.019-.081a.383.383 0 00-.033-.081.356.356 0 00-.318-.213.307.307 0 00-.15.035.443.443 0 00-.22.352l-.099.877a.156.156 0 01-.249.109.154.154 0 01-.063-.142l.099-.878a.754.754 0 01.389-.598.668.668 0 01.293-.07.667.667 0 01.6.389.822.822 0 01.081.262.033.033 0 010 .014zM21.834 20.389a.158.158 0 11-.314 0v-1.3a.131.131 0 010-.028.152.152 0 01.012-.051l.012-.028a.23.23 0 01.124-.103l.022-.005H21.748a.219.219 0 01.166.086l.837 1.07v-1.044a.157.157 0 11.315 0v1.3a.222.222 0 01-.115.193c-.006.005-.013.007-.019.01l-.013.006a.228.228 0 01-.17-.009l-.019-.011a.157.157 0 01-.032-.024c-.007-.006-.013-.014-.021-.02l-.004-.007-.839-1.075zM24.298 20.23a.159.159 0 010 .317h-.464a.622.622 0 01-.622-.621v-.475a.625.625 0 01.622-.622h.464c.122 0 .198.132.137.238a.163.163 0 01-.137.078h-.464a.308.308 0 00-.218.093.298.298 0 00-.093.216v.474c0 .17.138.308.308.308h.465zM25.211 19.142a.432.432 0 00-.299.118.394.394 0 00-.121.28v.294a.39.39 0 00.12.279.434.434 0 00.299.118.432.432 0 00.299-.118.389.389 0 00.122-.279v-.294a.394.394 0 00-.122-.28.437.437 0 00-.299-.118m0-.315a.742.742 0 01.517.206.701.701 0 01.217.506v.294a.696.696 0 01-.217.505.743.743 0 01-.517.207.74.74 0 01-.516-.207.702.702 0 01-.218-.505v-.294a.706.706 0 01.218-.506.74.74 0 01.516-.206zM18.565 19.143v1.087h.61a.197.197 0 00.145-.332l-.006-.006a.199.199 0 00-.139-.059h-.21a.158.158 0 01-.137-.236.157.157 0 01.137-.078h.071a.2.2 0 00.119-.057.185.185 0 00-.13-.318h-.46zm-.315.534v-.686c0-.021.003-.042.012-.061a.172.172 0 01.037-.055.154.154 0 01.103-.045h.618c.278 0 .503.226.503.502a.506.506 0 01-.077.268c.033.02.064.043.093.07l.009.012a.51.51 0 01.142.353.515.515 0 01-.513.512h-.77a.157.157 0 01-.157-.158z" fill="#1d1d1b"/></svg>
```

Comment:
Can you change the `aria-labelledby="pi-mb"` to `pi-multibanco` as suggested in the [Markup guideline](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#markup)?